ntoskrnl/io/iomgr/driver.c

```diff
@@ -1948,13 +1958,15 @@ IopLoadDriver(
     Status = IopGetRegistryValue(ServiceHandle, L"ImagePath", &kvInfo);
     if (NT_SUCCESS(Status))
     {
-        if ((kvInfo->Type != REG_EXPAND_SZ && kvInfo->Type != REG_SZ) || kvInfo->DataLength == 0)
+        if ((kvInfo->Type != REG_EXPAND_SZ && kvInfo->Type != REG_SZ) ||
+            (kvInfo->DataLength == 0) ||
```

Comment:
Based on your fixes in ntoskrnl/io/pnpmgr/devaction.c :
```suggestion
            (kvInfo->DataLength <= sizeof(WCHAR)) ||
```
(but maybe it should be strictly-less-than... ?)

Reply:
Yes. Fixed

Reply:
You used `<`, which "matches/improves" existing `== 0`.

For this PR or a follow-up:
Nonetheless, accepting an empty name seems potentially odd.
Especially, the checks in devaction.c refuse that, which looks like a potential inconsistency (here or there).

Reply:
I used `< sizeof(UNICODE_NULL)`, which matches the behavior of the existing code. At some point I would like to refactor this into a helper function, which should allow 0-sized strings.

Reply:
> refactor this

Filed as [CORE-20108](https://jira.reactos.org/browse/CORE-20108).